Editor/EditorCore/PolyShapeTool.cs

```diff
@@ -761,14 +761,13 @@ void HandleKeyEvent(Event evt)
                     {
                         DestroyImmediate(polygon.gameObject);
                         DestroyImmediate(this);
+                        ProBuilderEditor.ResetToLastSelectMode();
```

Comment:
This might not be necessary anymore, I'm not sure.

Reply:
This is not needed for the tool itself (move/rotate/scale) but is indeed needed to retrieve the previous PB mode (face/edge/vertex)